### PR TITLE
[stable10] Update changelog post sprint 7

### PR DESCRIPTION
**Pull request by PVince81**
Changelog update after [sprint 7](https://github.com/orgs/owncloud/projects/26?fullscreen=true)

@pmaier1

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
Looks good.

**Comment by phil-davis** [comment]
PR https://github.com/owncloud/core/pull/32512 "Support json for system:config:set" does not appear here. Maybe it was merged just after this changelog was done. But also, it is in the QA sprint project, not in the Dev sprint project. We should decide the process for changes/enhancements that are initiated by QA (because QA would like some functionality to help automated testing or...) but that also enhance or change the production system. Should those be put in the dev sprint project?

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,10 +42,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+- Minor dependency bumps 2018-08-26 - [#32439](https://github.com/owncloud/core/issues/32439)
+- Bump symfony 3.4.15 and zend-stdlib 3.2.1 - [#32499](https://github.com/owncloud/core/issues/32499)
 - Allow slashes in generated resource routes in app framework - [#31939](https://github.com/owncloud/core/issues/31939)
+- Email field is now default in user management page - [#32466](https://github.com/owncloud/core/issues/32466)
 
 ### Removed
 ### Fixed
+- Fix version previews to fall back to icon when no preview provider is available - [#32474](https://github.com/owncloud/core/issues/32474)
+- Fix master key recreation - [#32504](https://github.com/owncloud/core/issues/32504)
+- Return correct status when IMip email delivery fails - [#32489](https://github.com/owncloud/core/issues/32489)
```
Comment (patrickjahns) [inline CHANGELOG.md]: IMAP ?

Comment (phil-davis) [reply CHANGELOG.md]: There are mentions of:
```
 * iMIP handler.
 *
 * This class is responsible for sending out iMIP messages. iMIP is the
 * email-based transport for iTIP. iTIP deals with scheduling operations for
 * iCalendar objects.

class IMipPlugin extends SabreIMipPlugin {
```
so it seems to be "a thing".

**Comment by PVince81** [comment]
@phil-davis that PR hasn't been backported yet. The Changelog is based on the stable10 branch.

**Comment by phil-davis** [comment]
@PVince81 #32524 backport was merged about the same time you were making this changelog, so I guess that is why it "missed" the changelog. Not really a "miss" then.

**Pull request merged**